torch/nn/modules/activation.py

```diff
@@ -1170,9 +1170,9 @@ def forward(
             # generator expressions.
             if torch.overrides.has_torch_function(tensor_args):
                 why_not_fast_path = "some Tensor argument has_torch_function"
-            elif not all([_arg_cuda_or_cpu(x) for x in tensor_args]):
+            elif not all(_arg_cuda_or_cpu(x) for x in tensor_args):
```

Comment:
When did `torch.jit.script` get generator support? 0_0 . I thought this would fail tests for sure, hence why it's disabled in the latest flake8 config.

Reply:
I didn’t realize this was in torch.jit.script. I will disable the check instead

Reply:
@justinchuby I do not think it is, but I do think it should be torch.jit.script-able. Do we have any tests that actually exercise this though or did I miss some recent torch.jit.script update that gave generator support to it? Regardless, removing that particular rule from this initial PR should shrink the number of edits by quite a lot and make it a bit less controversial.

Reply:
@Skylion007 I added rudimentary generator support for TorchScript in https://github.com/pytorch/pytorch/pull/78142